src/course-page-resources/CoursePageResources.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import CoursePageConfigCard, { CoursePageShape } from './course-page/CoursePageConfigCard';
+
+import messages from './messages';
+
+function CoursePageResources({ intl, coursePages }) {
+  return (
+    <main>
+      <div className="container-fluid bg-info-100">
+        <div className="d-flex justify-content-between align-items-center border-bottom">
+          <h1 className="mt-3 text-info-500">{intl.formatMessage(messages.heading)}</h1>
+          <a className="btn btn-primary" href="/#" role="button">
```

Comment:
I was getting errors (linter, I think) without the `/`, otherwise I would have left it as the bare `#`

Reply:
@stvstnfrd This is the lint error I get without the `/`
`  14:11  error  The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid`

Reply:
@taniwha Okay, thanks. I wonder if `#/` works; I _think_ I've seen this in other React code :shrug: 

Otherwise, we don't need to worry about it for now.

We can consider revisting when we actually wire up behavior; if we don't use `href` directly, maybe we'll turn it into an actual button. But again, let's not change that yet.

Reply:
I've fixed this with the actual URL now.